radicale/utils.py

```diff
@@ -23,6 +23,9 @@
 
 def load_plugin(internal_types, module_name, class_name, configuration):
     type_ = configuration.get(module_name, "type")
+    if callable(type_):
```

Comment:
Plugins has clearly defined structure, plugins expect function interface consuming configparser's config and return an object implementing certain interface. When Radicale is used as standalone server it is perfectly okay for plugin loader to load plugins this way. But when you're not using plugin subsystem you should not enforce such behavior.